helm-swoop.el

```diff
@@ -515,7 +515,7 @@ If $linum is number, lines are separated by $linum"
             (let (($i 1))
               (insert (format "%s " $i))
               (while (re-search-forward "\n" nil t)
-                (cl-incf $i)
+                (setq $i (1+ $i))
```

Comment:
In fact they are the same, what's your point to change it?

Reply:
If we don't need to use the cl function, we should use pureElisp.
In fact, it is written this way elsewhere, and this is the only place where `cl-incf` is used.

Reply:
It was said that in old times emacser should avoid using `cl` because `cl` lacks prefix and pollute global namespace, but `cl-lib` had fixed this issue. So just feel free to use it.

I'm OK with these trivial code style changes. You can keep your choice.

Reply:
`cl-incf` is "pure Elisp", because `cl.el` is part of Emacs.  The function `cl-incf` alone is used over 300 times in the Emacs source code.  `cl-incf` is very clear about what it does: increments a variable.  `(setq $i (1+ $i))` is more complex and less clear.